pkg/csi/service/node.go

```diff
@@ -1069,6 +1092,32 @@ func getDevice(path string) (*Device, error) {
 	}, nil
 }
 
+func rescanDevice(ctx context.Context, dev *Device) error {
+	ctx = logger.NewContextWithLogger(ctx)
+	log := logger.GetLogger(ctx)
+
+	devRescanPath, err := getDeviceRescanPath(dev)
+	if err != nil {
+		return err
+	}
+
+	err = ioutil.WriteFile(devRescanPath, []byte{'1'}, 0666)
+	if err != nil {
+		mssg := fmt.Sprintf("Error rescanning block device %q. %v", dev.RealDev, err)
+		log.Error(mssg)
+		return fmt.Errorf(mssg)
```

Comment:
In this PR you're returning `fmt.Errorf`, `errors.New()` and `err.Error()` at different places. Can you consolidate all of it for consistency? I think `fmt.Errorf` is the most common one we use..

Reply:
Changed it wherever possible.